.eslintrc

```diff
@@ -0,0 +1,39 @@
+{
+    "env": {
+      "es6": true,
+      "browser": true,
+      "node": true,
+      "mocha": true
```

Comment:
Nice setup @edogbosunny but I think you should remove mocha(Line 6 of this file) since we are using Jest as our testing framework.

Reply:
Thanks @henperi this is being worked on.